Mg/dc 2429

molly-giddings opened the pull request.

molly-giddings commented:
@hiro-mishima - I had to delete my branch and recreate to get rid of unnecessary files that snuck into this PR.  Sorry about that!

hiro-mishima commented:
Hi @molly-giddings, it looks like `main.py` is still seen here as a changed file. It looks like your branch contains the commit from the other PR on top of the `develop` branch. In this type of situation, I usually create a new branch with a different name and make changes there to remove all the unnecessary commits. There must be more efficient ways but since the changes are not a lot here, it might not be a bad option. Please let me know if you need help with `git`!

hiro-mishima commented:
Hi @molly-giddings, it looks like `main.py` is still seen here as a changed file. It looks like your branch contains the commit from the other PR on top of the `develop` branch. In this type of situation, I usually create a new branch with a different name and make changes there to remove all the unnecessary commits. There must be more efficient ways but since the changes are not a lot here, it might not be a bad option. Please let me know if you need help with `git`!

molly-giddings commented:
Thanks, @hiro-mishima .  I deleted my previous branch both locally and remotely, did a rebase off of develop, and created a new branch.  From that, the linting check failed for main.py because of a ) that needed to be on the next line.  I corrected the line return and added main.py in to my new branch commit.  If you'd prefer a new-new branch :) I can repeat these steps.  Just let me know!

molly-giddings commented:
@hiro-mishima - I created a new branch called mg/dc_2429_b.  Maybe the previous one was still an issue despite deleting because it was the same name?
